pages/common/git-range-diff.md

```diff
@@ -0,0 +1,16 @@
+# git range-diff
+
+> Compare two commit ranges (e.g. two versions of a branch).
+> More information: <https://git-scm.com/docs/git-range-diff>.
+
+- Diff the changes in commit1 and commit2:
```

Comment:
Would this work?
```suggestion
- Diff the changes between two commits:
```

Reply:
I'm a bit wary here. Phrased like that it sounds like diff, which is what I'm trying to distinguish it from. It is not about the _changes between two commits_ but _the difference of changes within_ each commit (range).

Reply:
```suggestion
- Diff the changes in between 2 specific commits:
```

Reply:
Another suggestion:
```suggestion
- Diff the changes of two specific commits:
```

Reply:
@marchersimon your suggestion falls under the comment @xeruf made

Reply:
Then how about:
```suggestion
- Diff the changes made by two specific commits:
```
?

Reply:
Still think between is better but still need other's opinion

Reply:
What about this then?
```suggestion
- Diff the changes with in commit within the range from `commit1` to `commit2`:
```

Reply:
Yes that would work however its too long, my suggestion is still the best IMO

Reply:
`with in`??

Reply:
> Within and without should not be written with a space between with/in or with/out, according to all major dictionaries.

Reply:
I see.  Looks like a typo @sbrl.  I assume you mean like this?
```suggestion
- Diff the changes within the range from `commit1` to `commit2`:
```

Reply:
@bl-ue you are again misunderstanding the command. There is no range created of these two commits. Only the diffs of the commits themselves are diffed.

As for between: In all suggestions so far it could be interpreted also in its time dimension, feeding the same misunderstanding as @bl-ue

Reply:
@sbrl also understood it wrongly.
The suggestions by @marchersimon were pretty good so I adapted them.
Don't submit any more suggestions unless you are sure you know what the command does.

Reply:
I would be great if we could find an example... 😆

Reply:
Oops, my suggestion meant to be `within each commit`.